images/ubuntu-nix-systemd/Containerfile

```diff
@@ -8,20 +8,19 @@ LABEL org.opencontainers.image.licenses=MIT
 # Available versions in apt: https://packages.ubuntu.com/noble/curl
 # --no-install-recommends omits ca-certificates
 RUN apt-get update \
- && apt-get install --no-install-recommends -y curl	ca-certificates \
+ && apt-get install --no-install-recommends -y curl	ca-certificates xz-utils \
  systemd \
  git \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
 ENV PATH="${PATH}:/nix/var/nix/profiles/default/bin"
 
-RUN curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install linux \
-  --extra-conf "sandbox = false" \
-  --extra-conf "trusted-users = root user" \
-  --extra-conf "accept-flake-config = true" \
-  --no-start-daemon \
```

Comment:
As far as I see, similar option to this `no-start-daemon` is none in https://github.com/NixOS/nix